setup.py

```diff
@@ -49,6 +49,7 @@
     "google-cloud-storage >=2.0.0",
     "ibis-framework[bigquery] >=8.0.0,<9.0.0dev",
     "jellyfish >=0.8.9",
+    "numpy >=1.24.0",
```

Comment:
Does it help with the issue in the email? The user already had Numpy 1.24.0 in the issue they reported

Reply:
Yes. That's the whole issue. We weren't testing against this version so we broke them. Now we are.

Reply:
Ah, so the issue is with this particular version `numpy==1.24.0`, not necessarily all subsequent versions `numpy>=1.24.0`. E.g. colab comes with `numpy==1.26.4` and read_gbq works just fine there.